modules/configuring-external-databases/proc-configuring-default-configuration-for-secrets-and-pvcs.adoc

```diff
@@ -0,0 +1,71 @@
+:_mod-docs-content-type: PROCEDURE
+[id="proc-default-configuration-for-secrets-and-pvcs_{context}"]
+= Configuring default configuration for Secrets and PVCs
+
+By default, the mount path is the {product-short} container's working directory, and if you do not define the mount path, it defaults to `/opt/app-root/src`. The default configuration files are `secret-files` and PVCs. You can add the `rhdh.redhat.com/mount-path` annotation to your configuration file to specify the mount path.
```

Comment:
"The default configuration files are `secret-files` and PVCs." makes no sense.
Default configuration files types are listed in the upstream documentation (as previously mentioned).
'secret-files' and 'pvcs' are the types of default config files where annotations `rhdh.redhat.com/mount-path` and  'rhdh.redhat.com/containers' are applicable.